microcosm_postgres/encryption/store.py

```diff
@@ -81,3 +81,13 @@ def reencrypt_instance(self, instance, encryption_context_key):
         }
         values[self.model_class.__encryption_context_key__] = encryption_context_key
         return self.model_class(**values)
+
+    def search_encrypted_ids(self, context_id):
+        query = self.session.query(
+            self.model_class.id,
+        ).filter(
+            getattr(self.model_class, self.model_class.__encrypted_identifier__) != None,  # noqa
```

Comment:
`self.model_class.__encrypted_identifier__` just returns the name of the column; `self.model_class.encrypted_identifier` is similarly not a mapped column, but a property. Thus, we need to get the dynamic name of the column and `getattr` it for sqlalchemy to be map it.